# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1506099718 14400
#      Fri Sep 22 13:01:58 2017 -0400
# Node ID ca7cef297280f8138cea1175b4e214339a46bca3
# Parent  71a449b29cc7f5e4f99c6b99ba32248859d50874
fix another failing test after cset 1b7e49a72c62

* __osmesa_print__.cc: Change expected file size to new value in test.

diff --git a/libinterp/dldfcn/__osmesa_print__.cc b/libinterp/dldfcn/__osmesa_print__.cc
--- a/libinterp/dldfcn/__osmesa_print__.cc
+++ b/libinterp/dldfcn/__osmesa_print__.cc
@@ -219,17 +219,17 @@ instead.
 ##        For the moment, disable these tests on PC's and Macs.
 %!testif HAVE_OPENGL, HAVE_OSMESA, HAVE_GL2PS_H
 %! if (isunix ())
 %!   hf = figure ("visible", "off");
 %!   fn = tempname ();
 %!   unwind_protect
 %!     sombrero ();
 %!     __osmesa_print__ (hf, fn, "svg");
-%!     assert (stat (fn).size, 2938972, -0.20);
+%!     assert (stat (fn).size, 2183074, -0.20);
 %!     img = __osmesa_print__ (hf);
 %!     assert (size (img), [get(hf, "position")([4, 3]), 3]);
 %!     ## Use pixel sum per RGB channel as fingerprint
 %!     img_fp = squeeze (sum (sum (img), 2));
 %!     assert (img_fp, [52942515; 54167797; 56158178], -0.05);
 %!   unwind_protect_cleanup
 %!     close (hf);
 %!     unlink (fn);
